### PR TITLE
Added landmark element

### PR DESCRIPTION
**Pull request by NidhiKJha**
This pull request is to solve `Content is not within a landmark element`.

### CODE REVIEW

**Comment by NidhiKJha** [comment]
@josh-quoin @IanLawrence  Please review this pull request.

File: app/javascript/components/nav/components/menu-entry/component.jsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Component = ({
   const navItemName = name === "username" ? username : i18n.t(name);
 
   const renderNavAction = (
-    <div>
+    <li>
```
Comment (jtoliver) [inline app/javascript/components/nav/components/menu-entry/component.jsx:55]: Does this change how the menu looks? Same for the file above.

Comment (NidhiKJha) [reply app/javascript/components/nav/components/menu-entry/component.jsx:55]: I need to verify it again. I will do that

Comment (NidhiKJha) [reply app/javascript/components/nav/components/menu-entry/component.jsx:55]: I checked it and I couldn't see any change in the menu look

**Pull request merged**